core/src/de/adesso/brainysnake/BrainySnake.java

```diff
@@ -47,6 +54,12 @@ public void create() {
 
         texture = new Texture(pixmap);
 
+        try {
+            backgroundSound = Gdx.audio.newMusic(Gdx.files.internal("./core/src/de/adesso/brainysnake/sound/track.mp3"));
```

Comment:
Please move the audio file to the asset folder.
https://github.com/libgdx/libgdx/wiki/Managing-your-assets

rename track.mp3 to something more expressive

Reply:
Audio file renamed and moved to asset folder.